get_bams/1.0/get_bams.R

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/env Rscript
+#
+# Usage:
+#   get_bams.R <input_table> <output_table> [<library_id_column>]
+#
+# Notes:
+#   - <library_id_column> can be a name or the number 1. If it is the
+#     number 1, the script assumes that the input file has no header
+#     and only has a single column. The default value is 'library_id'
+#     (i.e., the script assumes a file header).
+#
+#   - The column names are prefixed with 'al.', 'lc.', and 'lb.' for
+#     designating alignment-, libcore-, and library-specific columns.
+#     It also helps with ensuring no collisions when joining the
+#     obtained information with the input table.
+#
+
+
+# Load packages -----------------------------------------------------------
+
+suppressWarnings(suppressPackageStartupMessages({
+
+message("Loading packages...")
+
+required_packages <- c("xml2", 
+              "getPass", 
+              "tidyverse", 
+              "httr", 
+              "jsonlite")
+
+for (pkg in required_packages){
+  if (!requireNamespace(pkg)) {
+    install.packages(pkg, repos = "https://mirror.rcg.sfu.ca/mirror/CRAN/")
+  }
+}
```

Comment:
I'm hesitant to have a script that messes with the environment. With the conda environment now added to this repo, do we think we still need this? We can leave it there, but comment it out.

Reply:
How does the conda environment work?

Reply:
It's the one you tested. The user just has to create a conda environment using the YAML file in this PR, and this script should work as expected.

Reply:
To be clear, I see the yaml file, what's the recommended usage?

Reply:
The user would create an environment using the usual `conda env` command. Then, they can either activate the environment before running the `Rscript` command or use the absolute path to the `Rscript` program (like you did). 
```
conda env create -n get_bams -f path/to/lcr-scripts/get_bams/1.0/get_bams-1.yaml
```

Reply:
I'm ambivalent. If the user chooses to create and activate the conda environment, the script won't install the packages because they're already installed. If they want to use it in their base environment with their default R install, it will install any missing packages for them in their default library location. Maybe leaving it in is the best of both worlds?

Reply:
Is it possible to have an interactive prompt after the `if (!requireNamespace(pkg))` condition, where if the packages are missing users can be prompted to either quit and install/activate the conda environment, or proceed with installing the required packages in the current environment?

Reply:
I didn't consider that the command wouldn't be run if the packages are installed already. That makes it a bit better. I still wouldn't want a script to change my environment without my consent. A prompt asking the user would be a good compromise. There are ways of doing this in R, but [this SO post](https://stackoverflow.com/a/44013192/1615670) presents a potential issue if not done in a proper way. 

If you feel like quickly implementing the prompt, that would be ideal, but I'm not going to hold back this PR for this. Feel free to merge the PR if you don't think it's necessary. If someone complains about this down the road, it can be addressed then.